omods/core/api/src/main/java/org/openmrs/module/icare/billing/services/insurance/nhif/claim/FolioItem.java

```diff
@@ -193,7 +192,7 @@ public void setLastModified(Date lastModified) {
 	public String itemCode;
 	
 	@JsonProperty("ItemName")
-	private String itemName;
+	public String itemName;
```

Comment:
Why did you update the access modifier to public while the property can still be accessed by its getter and setter methods?